login-script.php

```diff
@@ -4,33 +4,42 @@
     include("./classes/functions.php");
 
     $email = sanitize($_POST["email"]);
-    // $passwd = sanitize($_POST["password"]);
+    $passwd = sanitize($_POST["passwd"]);
+    // $pass = $sql = "SELECT * from `password` WHERE `passwd` = '$passwd'";
 
     $sql = "SELECT * from `password` WHERE `email` = '$email'";
+    // $passwdDB = "SELECT * from `password` WHERE `passwd` = '$passwd'";
+    
+    
     $result = mysqli_query($conn, $sql);
-
+    $record = mysqli_fetch_assoc($result);
+    $encpass = $record["passwd"];
     if (mysqli_num_rows($result) == 1){
-        $record = mysqli_fetch_assoc($result);
-        echo("Successful login");
-        $sql = "SELECT * FROM `klant` WHERE `email` = '$email'";
-        $query = mysqli_query($conn, $sql);
-        $data = mysqli_fetch_assoc($query);
-        $userrole = "klant";
-
-        //declare new variables to show user login
-        session_start();
-        $_SESSION["email"] = $record["email"];
-        $_SESSION["logged_in"] = true;
-        $_SESSION["userrole"] = $userrole;
-        // $_SESSION["voornaam"] = $data["voornaam"];
-        // $_SESSION["achternaam"] = $data["achternaam"];
-        setcookie("email", $record["email"], time()+3600);  /* expire in 1 hour */
-        setcookie("logged_in", true, time()+3600);  /* expire in 1 hour */
-        setcookie("userrole", $userrole, time()+3600);  /* expire in 1 hour */
-        // $_SESSION["rol"] = $record["rol"];
-        header("Location:./customer.php");
+        if(password_verify($passwd, $encpass)) {
```

Comment:
if (password_verify($passwd.$salt, $encpass)) {success}

In the register process, the hashed password is created out of the following:

Passwd(user input)+salt(random 10 letter string)